package-system/astc-encoder/install_mac.sh

```diff
@@ -17,8 +17,8 @@ cp -f temp/src/Source/astcenc.h $INCLUDE_PATH/ || exit $?
 
 BUILD_PATH=temp/build/Source
 
-# copy static lib and executable
-cp -f $BUILD_PATH/astcenc-native $BIN_PATH/ || exit $?
-cp -f $BUILD_PATH/libastcenc-native-static.a $BIN_PATH/ || exit $?
+# copy sse2 lib and executable
```

Comment:
comment says sse2, should be sse4?

Reply:
nice catch.  I can update this without invalidating the packages, thankfully